cluster/mesos/README.md

```diff
@@ -8,33 +11,48 @@ containers on a Mesos cluster.
 
 ## Prerequisites
 
-Each node in your swarm must run a Mesos slave. The slave must be capable of starting tasks in a Docker Container using the `--containerizer=docker` option.
+Each node in your swarm must run a Mesos slave. The slave must be capable of
+starting tasks in a Docker Container using the `--containerizer=docker` option.
 
-You need to configure two TCP ports on the slave. One port to listen for the swarm manager, for example 2375. And a second TCP port to listen for the Mesos master, for example 3375.
+You need to configure two TCP ports on the slave. One port to listen for the
+swarm manager, for example 2375. And a second TCP port to listen for the Mesos
+master, for example 3375.
 
 ## Start the Docker Swarm manager
 
 If you use a single Mesos master:
 
 ```     
-docker run -d -p <swarm_port>:2375 -p 3375:3375 swarm manage -c mesos-experimental --cluster-opt mesos.address=<public_machine_ip> --cluster-opt mesos.port=3375 <mesos_master_ip>:<mesos_master:port>
+$ docker run -d -p <swarm_port>:2375 -p 3375:3375 \
+    swarm manage \
+        -c mesos-experimental \
+        --cluster-opt mesos.address=<public_machine_ip> \
+        --cluster-opt mesos.port=3375 \
+        <mesos_master_ip>:<mesos_master:port>
 ```
 
-If you use multiple Mesos masters:
-
-```
-docker run -d -p <swarm_port>:2375 -p 3375:3375 swarm manage -c mesos-experimental --cluster-opt mesos.address=<public_machine_ip> --cluster-opt mesos.port=3375 zk://<mesos_masters_url>
-```
+The command above creates a swarm manager listening at `<swarm_port>` and
```

Comment:
Deconstruct these compound sentences to make it easier to read:

--->

The command above creates a swarm manager listening at `<swarm_port>`. The `<mesos_master_ip>` value points to where Mesos master lives in the cluster. Typically, this is localhost, a hostname, or an IP address.  The `<public_machine_ip>` value is the IP address for Mesos master to talk to swarm manager. If master and manager are co-located on the same machine, you can use the `0.0.0.0` or `localhost` value.